### ISSUE TITLE
Stanford CoreNLP 3.8 not compatible 

### ISSUE THREAD

**Issue by bluemonk482** [abisee/cnn-dailymail#5]
Thanks for the code abisee.

I have tried to use the new stanford corenlp 3.8, but it seems it can only tokenise the first data file aka the first line in mapping.txt and thus:

```
Exception in thread "main" java.io.IOException: Stream closed
	at java.io.BufferedWriter.ensureOpen(BufferedWriter.java:116)
	at java.io.BufferedWriter.write(BufferedWriter.java:221)
	at java.io.Writer.write(Writer.java:157)
	at edu.stanford.nlp.process.PTBTokenizer.tokReader(PTBTokenizer.java:505)
	at edu.stanford.nlp.process.PTBTokenizer.tok(PTBTokenizer.java:450)
	at edu.stanford.nlp.process.PTBTokenizer.main(PTBTokenizer.java:813)
```

Corenlp 3.7 works perfectly though. Jus thought you might want to know this.

**Comment by kaihuchen** [comment]
I am observing exactly the same problem.

Where can I find the official version of the Corenlp 3.7 kit?

**Comment by JafferWilson** [comment]
@kaihuchen Please can yoou tell me what jar file you have used ... Stanford library 2016 or 2017 one?
Please let me know. If you are using the 2017 one then this issue will persist. You have to use the 2016 one. Hope this will help you.

**Comment by kaihuchen** [comment]
@JafferWilson  Yes, indeed I was using the 2017-06-09 Stanford library. I managed to find the 2016-10-31 version on the stanford v.3.7.0 Corenlp kit (which was not explicitly listed on their website at https://stanfordnlp.github.io/CoreNLP/, but actually is accessible with **wget http://nlp.stanford.edu/software/stanford-corenlp-full-2016-10-31.zip**). After I switched to the 2016 version (v 3.7.0) it now runs fine. Thanks for your help.
